### PR TITLE
Pms v15 multinetwork

### PR DESCRIPTION
**Pull request by frederikgeth**
1065 passed, 91 failed, 28 errored, 0 broken.

### CODE REVIEW

**Comment by frederikgeth** [comment]
1069 passed, 98 failed, 22 errored

File: src/core/variable_mx.jl [hunk 1]
```diff
@@ -68,7 +68,7 @@ function variable_mx_real(model::JuMP.Model, indices::Array{T,1}, N::Int, M::Int
     dict_mat_vars_temp = Dict{T, Array{Any, 2}}([(i,fill(missing,N,M)) for i in indices])
     for n in 1:N
         for m in 1:M
-            varname = isempty(prefix) ? "$(n)$(m)_$name" : "$(prefix)_$(n)$(m)_$name"
+            varname = isempty(prefix) ? "$(name)_$(n)$(m)" : "$(prefix)_$(n)$(m)"
```
Comment (sanderclaeys) [inline src/core/variable_mx.jl]: `$(prefix)_$(n)$(m)` should be `$(prefix)_$(name)_$(n)$(m)`

prefix is usually the network number

Comment (sanderclaeys) [reply src/core/variable_mx.jl]: correct in the other functions though, typo probably

Comment (frederikgeth) [reply src/core/variable_mx.jl]: thanks, changed now

**Pull request merged**